Makefile

```diff
@@ -1,4 +1,4 @@
-.PHONY: clean clean-mocks coveralls testpop lint mocks out setup test test-integration test-unit test-race
+.PHONY: clean clean-mocks coveralls testpop lint mocks out setup test test-integration test-unit test-race test-all
```

Comment:
I wanted to have an easy command to run all tests between changes to make sure I didn't have regressions (caught a couple of races before pushing) but there wasn't an easy command so I added it. Could be done in its own PR but would make it more complicated to land.

If you don't want it in here I can remove it but I thought it might be useful to have in the future.